android-components/components/tooling/lint/src/main/java/mozilla/components/tooling/lint/ImageViewAndroidTintXmlDetector.kt

```diff
@@ -36,7 +35,6 @@ class ImageViewAndroidTintXmlDetector : ResourceXmlDetector() {
         const val ERROR_MESSAGE =
             "Using android:tint to tint ImageView instead of app:tint with AppCompatImageView"
 
-        @VisibleForTesting
```

Comment:
When doing a checksum diff between Fenix and the AC component, this was a difference and didn't need to be there. Removing it to ensure that when BZ-1872229 is worked on, its a tiny bit easier for the next person to de-duplicate and save them the trouble.